Dockerfile

```diff
@@ -192,11 +192,10 @@ RUN pip install --user pipenv
 
 # Poetry
 
-RUN curl -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py | python
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
```

Comment:
Shouldn't we install a specific version like other tools?

```suggestion
RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python --version 1.0.0
```

Of cause `1.0.0` should com from env

Reply:
Won't work, it will print python version

Reply:
Can we do like in the dedicated docker image and get it then run it?

Reply:
See updated suggestion, should work added the `-` to force `stdin` mode